.changeset/long-cups-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Fixed Docker compatibility issues by upgrading ast-grep.
```

Comment:
Updated `ast-grep` to `0.19.3` which fixed support for running Ponder on Alpine Linux-based Docker images.